tests/e2e_testdata.go

```diff
@@ -0,0 +1,514 @@
+			}
+		},
+		"gatewayAAT": {
+			"address": "test_address_8dbb89278918da056f589086fb4",
+			"applicationPublicKey": "%s",
+			"applicationSignature": "test_key_f9c21a35787c53c8cdb532cad0dc01e099f34c28219528e3732c2da38037a84db4ce0282fa9aa404e56248155a1fbda789c8b5976711ada8588ead5",
+			"clientPublicKey": "test_key_2381d403a2e2edeb37c284957fb0ee5d66f1081acb87478a5817919",
+			"privateKey": "test_key_0c0fbd26d98bcbdca4d4f14fdc45ffb6db0e6a23a56671fc4b444e1b8bbd4a934adde984d117f281867cb71d9537de45473b3028ead2326cd9e27ad",
+			"version": "0.0.1"
+		},
+		"gatewaySettings": {
+			"secretKey": "test_key_ba2724be652eca0a350bc07",
+			"secretKeyRequired": false
+		},
+			"notificationSettings": {
+			"signedUp": true,
+			"quarter": false,
+			"half": false,
+			"threeQuarters": true,
+			"full": true
+		}
+	}`
+	loadBalancerJSON = `{
+		"name": "test-load-balancer-%d",
+		"userID": "%s",
+		"applicationIDs": ["%s"],
+		"requestTimeout": 2000,
+		"gigastake": false,
+		"gigastakeRedirect": true,
+		"stickinessOptions": {
+			"duration": "",
+			"stickyOrigins": null,
+			"stickyMax": 0,
+			"stickiness": false
+		},
+		"Applications": null
+	}`
+)
+
+/* Test Utility Funcs */
+
+// Initializes the Influx client and returns it alongside the org ID
+func initTestInfluxClient(c *require.Assertions) (influxdb2.Client, string) {
+	client := influxdb2.NewClientWithOptions(
+		influxOptions.URL, influxOptions.Token,
+		influxdb2.DefaultOptions().SetBatchSize(4_000),
+	)
+
+	dOrg, err := client.OrganizationsAPI().FindOrganizationByName(ctx, influxOptions.Org)
+	c.NoError(err)
+
+	health, err := client.Health(ctx)
+	c.NoError(err)
+	c.Equal("ready for queries and writes", *health.Message)
+
+	return client, *dOrg.Id
+}
+
+// Resets the Influx bucket each time the collector test runs
+func resetInfluxBuckets(client influxdb2.Client, orgID string, c *require.Assertions) {
+	bucketsAPI := client.BucketsAPI()
+	usedBuckets := []string{mainBucket, main1mBucket, influxOptions.CurrentBucket, influxOptions.CurrentOriginBucket}
+
+	for _, bucketName := range usedBuckets {
+		bucket, err := bucketsAPI.FindBucketByName(ctx, bucketName)
+		if err == nil {
+			err = client.BucketsAPI().DeleteBucketWithID(ctx, *bucket.Id)
+			c.NoError(err)
+		}
+		_, err = client.BucketsAPI().CreateBucketWithNameWithID(ctx, orgID, bucketName)
+		c.NoError(err)
+	}
+}
+
+// Initializes the Influx tasks used to populate each bucket from the main bucket
+func initInfluxTasks(tasksAPI influxAPI.TasksAPI, orgID string, c *require.Assertions) []*influxDomain.Task {
+	app1mString := fmt.Sprintf(app1mStringRaw, mainBucket, todayFormatted, endOfDayFormatted, main1mBucket)
+	app10mString := fmt.Sprintf(app10mStringRaw, main1mBucket, todayFormatted, endOfDayFormatted, influxOptions.CurrentBucket)
+	origin60mString := fmt.Sprintf(origin60mStringRaw, mainBucket, todayFormatted, endOfDayFormatted, influxOptions.CurrentOriginBucket)
+	taskNames := []string{"app-1m", "app-10m", "origin-sample-60m"}
```

Comment:
Perhaps all the above could be stored in a map instead to better represent their relationship, i.e. task name and task contents?

Reply:
Sure that makes sense, I will update.